tests/unit/scoring/test_metcalf_scoring.py

```diff
@@ -194,10 +280,198 @@ def test_get_links_invalid_mixed_types(mc, spectra, mfs):
     assert ".Spectrum" in str(e.value)
 
 
-def test_get_links_no_linkfinder(npl, gcfs):
-    """Test `get_links` method when no LinkFinder object is found."""
-    mc = MetcalfScoring(npl)
-    mc.LINKFINDER = None
+#
+# Test the `_get_links` method
+#
+
+
+def test__get_links_gcf(mc, datalinks, gcfs):
+    """Test `get_links` method for input GCF objects."""
+    mc.calc_score(datalinks, link_type="spec-gcf")
+    mc.calc_score(datalinks, link_type="mf-gcf")
+    index_names = ["source", "target", "score"]
+
+    # cutoff = negative infinity (float)
+    links = mc._get_links(*gcfs, score_cutoff=np.NINF)
+    assert len(links) == 2
+    # expected values got from `test_calc_score_raw_score`
+    assert_frame_equal(
+        links[0],
+        pd.DataFrame(
+            [
+                ["gcf1", "gcf2", "gcf3"] * 3,
+                [
+                    *["spectrum1"] * 3,
```

Comment:
It's unpacking the list `["spectrum1", "spectrum1", "spectrum1",]` to `"spectrum1", "spectrum1", "spectrum1"`. It's needed, otherwise what we get is a list.